src/lib/Components/FilterModal.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, Button, CloseIcon, Flex, Sans } from "@artsy/palette"
+import React from "react"
+import { LayoutAnimation, Modal as RNModal, TouchableWithoutFeedback, ViewProperties } from "react-native"
+import styled from "styled-components/native"
+
+interface ModalProps extends ViewProperties {
+  visible?: boolean
+  closeModal?: () => void
+}
+
+interface State {
+  isComponentMounted: boolean
+}
+
+const ModalBackgroundView = styled.View`
+  background-color: #00000099;
+  flex: 1;
+  flex-direction: column;
+`
+
+const ModalInnerView = styled.View<{ visible: boolean }>`
+  flex-direction: column;
+  width: 100%;
+  background-color: white;
+  height: ${({ visible }) => (visible ? "auto" : "0")};
+  padding: ${({ visible }) => (visible ? "20px" : "0")};
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+`
+
+export class FilterModal extends React.Component<ModalProps, State> {
+  constructor(props) {
+    super(props)
+
+    this.state = { isComponentMounted: false }
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut)
+      this.setState({ isComponentMounted: true })
+    }, 100)
+  }
+
+  closeModal() {
+    if (this.props.closeModal) {
+      this.props.closeModal()
+    }
+  }
+
+  render() {
+    /*
+    https://app.zeplin.io/project/5aabc5e0786bbb29b6e3dc7f/screen/5dba03df09bf3f5547162ac9
+
+    Next steps:
```

Comment:
#minor but given we've tasked these out in Jira I don't think you need to commit these "Next Steps" comments into code!